taskRequests/README.md

```diff
@@ -0,0 +1,15 @@
+# Task Request
+
+Firestore Task Requests collection data model
+
+```
+id: <id>,
+taskId: <id>,
+title: string,
+requestedBy:  [string],
+purpose: string,
+type: string,
+isNoteWorthy: boolean,
+currentStatus: string,
```

Comment:
`status` will have a fixed set of values. 
So, instead of `string` here use set of statuses values here.